test/logout_responses/logoutresponse_fixtures.rb

```diff
@@ -8,8 +8,8 @@ def default_response_opts
   }
 end
 
-def valid_response(opts = {})
-  opts = default_response_opts.merge!(opts)
+def valid_logout_response_document(opts = {})
+  opts = default_logout_response_opts.merge!(opts)
```

Comment:
do we need to alter `default_logout_response_opts`? it's being modified when using `merge!`